### ISSUE TITLE
Core Exception with DCE-elements

### ISSUE THREAD

**Issue by subnoodle** [a-r-m-i-n/dce#47]
**Versions**
- DCE version: **2.9.3**
- TYPO3 version: **11.5.23**
- PHP version: **8.1.12**

**Describe the bug**
We get the error message:

Core: Exception handler (WEB): Uncaught TYPO3 Exception: #1316104317: The default controller for extension "Dce" and plugin "dceuid7" can not be determined. Please check for TYPO3\CMS\Extbase\Utility\ExtensionUtility::configurePlugin() in your ext_localconf.php. | TYPO3\CMS\Extbase\Mvc\Exception thrown in file /html/typo3/web/typo3/sysext/extbase/Classes/Mvc/Web/RequestBuilder.php in line 217. Requested URL: https://dummy-stage.domain.tld/a-slug

Did anyone get this error too?

We upgraded the typo3-instance two week before to typo3 v11. The first site on production runs well, buit the 2nd (multisite) has some problem as well. Beneath the error message some elements do not get rendered at all and stay blank.

The rendered html-code in debug of the fuild template looks very strange:
```
   content => ' <div id="c17568" class="frame frame-default frame-type-header frame-layout-0">
       <header class=page-header> <h3 class=""> <p>Something aaa</p> </h3> </header>     
       </div> <div id="c" class="frame frame- frame-type- frame-layout-">
       </div><div id="c" class="frame frame- frame-type- frame-layout-">
       </div>' 
```
Please look at the part of _div id="c"_  - already the uid is missing.
No wonder, since the messages tells it cannot even setup the plugin: The whole content is missing as well.

Did anyone experience that issue already? How to solve?
We've seen a part of the code in
Classes\Components\ContentElementGenerator\OutputPlugin.php:62
asking for Typo3-Version10.
Should this and other locations with similar code be expanded to Typo3 v11? (Maybe i'm totally wrong)

We are looking forward for any solution.

Thank you, Thomas

**Comment by a-r-m-i-n** [comment]
Is this issue still occurring, in latest DCE version?

**Comment by miko007** [comment]
happens to me on first page load after clearing cache on TYPO3 `v11.5.27` and DCE `v2.9`

**Comment by a-r-m-i-n** [comment]
@miko007 This is super weird. I'm curious about the circumstance, that the second request works.

Probably there is an issue in one or more DCE configurations, which leads to issues in cached results. Is the DCE cache (two files) written? And if yes, can you spot some issue in generated PHP code? Some Syntax error maybe. Or interrupted code.

If you can not spot the issue in the cache files, please provide them as gist, so I can check.

**Comment by a-r-m-i-n** [comment]
I think this issue is caused because of old controller name ("dce") in DCE php output.
I've fixed this in #61 

Please report, if this issue still occure.
